plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftMetastoreUtil.java

```diff
@@ -679,10 +679,9 @@ private static void fromMetastoreApiStorageDescriptor(
                 .setSerdeParameters(serdeInfo.getParameters() == null ? ImmutableMap.of() : serdeInfo.getParameters());
     }
 
-    private static StorageDescriptor makeStorageDescriptor(String tableName, List<Column> columns, Storage storage)
+    private static StorageDescriptor makeStorageDescriptor(List<Column> columns, Storage storage)
     {
         SerDeInfo serdeInfo = new SerDeInfo();
-        serdeInfo.setName(tableName);
```

Comment:
`name` isn't an optional field at https://github.com/apache/hive/blob/04bb92ae6174d2953f42801593b1b709bcf1154c/standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift#L443. Is this change really safe?

Reply:
I find no where using this field. Spark doesn't set name either ([code](https://github.com/apache/spark/blame/339d1c9d9d50bce63316ac788626bea998e71b06/sql/hive/src/main/scala/org/apache/spark/sql/hive/client/HiveClientImpl.scala#L1186))